core/packages/test/config/parachain-relay.json

```diff
@@ -1,28 +1,27 @@
 {
-    "source": {
-        "ethereum": {
-            "endpoint": "ws://127.0.0.1:8546"
-        },
-        "polkadot": {
-            "endpoint": "ws://127.0.0.1:9944"
-        },
-        "parachain": {
-            "endpoint": "ws://127.0.0.1:11144"
-        },
-        "contracts": {
-            "BeefyClient": null,
-            "BasicInboundChannel": null
-        },
-        "beefy-activation-block": 0,
-        "basicChannelSourceIDs": []
+  "source": {
+    "ethereum": {
+      "endpoint": "ws://127.0.0.1:8546"
     },
-    "sink": {
-        "ethereum": {
-            "endpoint": "ws://127.0.0.1:8546",
-            "gas-limit": 5000000
-        },
-        "contracts": {
-            "BasicInboundChannel": null
-        }
+    "polkadot": {
+      "endpoint": "ws://127.0.0.1:9944"
+    },
+    "parachain": {
+      "endpoint": "ws://127.0.0.1:11144"
+    },
+    "contracts": {
+      "BeefyClient": null,
+      "BasicInboundChannel": null
+    },
+    "beefy-activation-block": 0,
+    "basicChannelSourceIDs": []
+  },
+  "sink": {
+    "ethereum": {
+      "endpoint": "ws://127.0.0.1:8546"
```

Comment:
`gas-limit` was removed. Don't we need `"gas-limit": null` here, like in [`beefy-relay.json`](https://github.com/Snowfork/snowbridge/pull/777/files/dd12d04c34f67438160824ed77965e6444b6067b..6b9c725f04fa0736ee2b8076ec4e15358649d8f1#diff-7073e2b92ca88949ecf2c3fcbfe1ddfbdcdecafb89901f7689b78310ffa35099)?

Or could we remove the `gas-limit` line there?

Reply:
Yes, we need to add it. Test shows still required for local setup.